apps/frontend/components/FormValidation/FloodFormValidation/tablesConfig/NumAffected-1.tsx

```diff
@@ -3,75 +3,80 @@ import { FLOOD, FloodSpecific } from '@wfp-dmp/interfaces';
 
 import { getColumnSetup, getGroupSetup } from 'utils/tableFormatting';
 
-export const NumAffected1Columns: GridColDef[] = [
-  getColumnSetup(FloodSpecific.NumVillAff, FLOOD),
-  getColumnSetup(FloodSpecific.NumPeoAff, FLOOD),
-  getColumnSetup(FloodSpecific.NumFamAff, FLOOD),
-  getColumnSetup(FloodSpecific.NumFeAff, FLOOD),
-  getColumnSetup(FloodSpecific.NumTDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumMeDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumFeDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumKidDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumOldDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumDisDeath, FLOOD),
-  getColumnSetup(FloodSpecific.NumTMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumMeMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumFeMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumKidMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumOldMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumDisMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumTInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumMeInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumFeInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumKidInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumOldInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumDisInjure, FLOOD),
+// TODO - Finish reorganizing this file for typing
+const NumAffected1ColumnsKeys: FloodSpecific[] = [
+  'NumVillAff',
+  'NumPeoAff',
+  'NumFamAff',
+  'NumFeAff',
+  'NumFeAff',
+  'NumTDeath',
+  'NumMeDeath',
+  'NumFeDeath',
+  'NumKidDeath',
+  'NumOldDeath',
+  'NumDisDeath',
+  'NumTMissing',
+  'NumMeMissing',
+  'NumFeMissing',
+  'NumKidMissing',
+  'NumOldMissing',
+  'NumDisMissing',
+  'NumTInjure',
+  'NumMeInjure',
+  'NumFeInjure',
+  'NumKidInjure',
+  'NumOldInjure',
+  'NumDisInjure',
 ];
 
+export const NumAffected1Columns: GridColDef[] = NumAffected1ColumnsKeys.map(
+  key => getColumnSetup(key, FLOOD),
+);
+
+const keyAsField = (key: FloodSpecific) => ({ field: key });
+const keysAsChildren = (keys: FloodSpecific[]) => keys.map(keyAsField);
+
 export const NumAffected1ColumnGroup: GridColumnGroupingModel = [
   {
     ...getGroupSetup('victimsAffected', FLOOD),
     children: [
-      { field: FloodSpecific.NumVillAff },
+      keyAsField('NumVillAff'),
       {
         ...getGroupSetup('totalAffected', FLOOD),
-        children: [
-          { field: FloodSpecific.NumPeoAff },
-          { field: FloodSpecific.NumFamAff },
-          { field: FloodSpecific.NumFeAff },
-        ],
+        children: keysAsChildren(['NumPeoAff', 'NumFamAff', 'NumFeAff']),
       },
       {
         ...getGroupSetup('deathToll', FLOOD),
         children: [
-          { field: FloodSpecific.NumTDeath },
```

Comment:
I think we should continue to use the string Enum to avoid any typos

Reply:
This offers a check against typos as well. I proposed a few other option in the PR body. We can chat about it but it's not urgent anyway.